hq/test/schedule/DynamoTest.scala

```diff
@@ -0,0 +1,114 @@
+package schedule
+
+import com.amazonaws.auth.{AWSStaticCredentialsProvider, BasicAWSCredentials}
+import com.amazonaws.client.builder.AwsClientBuilder.EndpointConfiguration
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder
+import com.amazonaws.services.dynamodbv2.model._
+import model._
+import org.joda.time.DateTime
+import org.scalatest.{BeforeAndAfterAll, FreeSpec, Matchers}
+
+import scala.collection.JavaConverters._
+
+
+class DynamoTest extends FreeSpec with BeforeAndAfterAll with Matchers with AttributeValues {
+
+  private val client = {
+    val conf = new EndpointConfiguration("http://localhost:8000", config.Config.region.name)
+    AmazonDynamoDBClientBuilder.standard
+      .withEndpointConfiguration(conf)
+      .withCredentials(new AWSStaticCredentialsProvider(new BasicAWSCredentials("dummy", "credentials")))
+      .build
+  }
+
+  private val tableName = "security-hq-iam-TEST"
+  private val tableAttributes = Seq((Symbol("id"), ScalarAttributeType.S))
+  private val keySchemaAttributes = tableAttributes
+  private val arbitraryThroughputThatIsIgnoredByDynamoDBLocal = new ProvisionedThroughput(1L, 1L)
+
+  override def beforeAll () {
```

Comment:
Is this boilerplate?

Reply:
If you mean what's `beforeAll` for - it's the way you tell scalatest to run some side effects before any of your tests. There's also `afterAll` and `beforeEach`/`afterEach` for similar.

Reply:
Oh sorry, I didn't really make it obvious what I was referring to. I meant the implementation of `beforeAll`. Specifically I was wondering if `attributeDefinitions` and `keySchema` were boilerplate picked up from documentation, or if it was your own implementation

Reply:
Ah yes, it is quite unpleasant. Mostly taken from https://github.com/scanamo/scanamo/blob/master/testkit/src/main/scala/org/scanamo/LocalDynamoDB.scala